01_literature/Engineering/Frontend/frontend-report-september-2024.md

```diff
@@ -0,0 +1,122 @@
+
+Prominent figures in the JavaScript world are calling on Oracle to release the "JavaScript" trademark: *You have long ago abandoned the JavaScript trademark, and it is causing widespread, unwarranted confusion and disruption.*
+
+### [Bytes #323 - Hono: The Next Big Thing in Web Frameworks?](https://bytes.dev/archives/323)
+
+Discover why it's causing a stir in the JavaScript world. Think blazing-fast speeds and effortless full-stack development. And some other cool bits.
+
+### Quick Links
+
+- [Prompt design - A case for building better tools for it](https://www.cursor.com/blog/prompt-design)
+- [Using ChatGPT to reverse engineer minified JavaScript](https://glama.ai/blog/2024-08-29-reverse-engineering-minified-code-using-openai)
+
+## New Tools
+
+### [Meet Conform: The Type-Safe Form Validation Library](https://conform.guide/)
+
+Conform is a type-safe form validation library utilizing web fundamentals to progressively enhance HTML Forms with full support for server frameworks like [Remix](https://remix.run/) and [Next.js](https://nextjs.org/).
+
+### [NewDevTools: Your Daily Dose of Developer Goodies](https://newdevtools.com/)
+
+Discover new developer tools & services daily with NewDevTools. Updated daily!
+
+### [Bulletproof React - A simple, scalable, and powerful architecture for building production ready React applications](https://github.com/alan2207/bulletproof-react)
+
+This is not supposed to be a template, boilerplate or a framework, but more of an opinionated guide, showcasing solving most of the real-world problems of an application in a practical way and help developers write better applications.
+
+### Quick Links
+
+- [Tailwind CSS for Emails? Now You Can Inline Those Styles!]([tailwind-to-inline - npm (npmjs.com)](https://www.npmjs.com/package/tailwind-to-inline))
+- [Vaul 1.0: New React Drawer Component]([Vaul (emilkowal.ski)](https://vaul.emilkowal.ski/))
+- [DMNO - Environment Variables Evolved](https://dmno.dev/)
+- [prabhuignoto/react-chrono: 🕑 Modern Timeline Component for React](https://github.com/prabhuignoto/react-chrono)
+- [Write and Code with ChatGPT: Introducing Canvas](https://openai.com/index/introducing-canvas/)
+
+## Commentary
+
+- [I am tired of AI](https://www.ontestautomation.com/i-am-tired-of-ai/)
+- [Legacy Modernization meets GenAI](https://martinfowler.com/articles/legacy-modernization-gen-ai.html)
+- [Why Copilot is Making Programmers Worse at Programming](https://www.darrenhorrocks.co.uk/why-copilot-making-programmers-worse-at-programming/)
+- [Why Gumroad Didn't Choose htmx](https://htmx.org/essays/why-gumroad-didnt-choose-htmx/)
+- [The hardest bug investigation of my career and the insane code that caused it](https://www.reddit.com/r/ExperiencedDevs/comments/1fu0e5q/the_hardest_bug_investigation_of_my_career_and/)
+- [Why Not Comments - Why not "why not" comments? Not why "not comments"](https://buttondown.com/hillelwayne/archive/why-not-comments/)
```

Comment:
IMO, only the htmx link is related to the frontend

Reply:
Some I find are worth reading for its engineering aspect. These are just commentaries anyway 🤔